contrib/kind.sh

```diff
@@ -1163,5 +1167,9 @@ if [ "$KIND_INSTALL_PLUGINS" == true ]; then
 fi
 if [ "$KIND_INSTALL_KUBEVIRT" == true ]; then
   install_kubevirt
+  deploy_primary_udn_nad
+  deploy_passt_binary
+fi
+if [ "$KIND_INSTALL_IPAM_CONTROLLER" == true ]; then
```

Comment:
a by product is that whoever not installs ipam wont get cert-manager and would need to install it itself

i.e ipam-extensions

unless the cert-manager would moved as part of kubevirt

Reply:
yes. The way I see it, only ipam-ext needs cert manager - so it's OK to only deploy cert-mgr if ipam-ext is requested.
If someone wants to deploy ipam-ext out of band (like @maiqueb) then they will have to also install cert manager themsleves.
The important thing is that e2e tests use the official ipam (with the official supported cert-manager for ipam).

Reply:
it will break on ipam-extensions
this is an action item that need to be solved please, this what i meant

Reply:
ah I got you then. If I understand what your mean - then it's not related to this PR right?

well we can decide to include that cert-manager manifest on the ipam-repo, then you could align them here, possibly diverging them.

Reply:
It is part of the task you are on, i dont see why it should be delegated to someone else

Reply:
Maybe that's the correct thing to do - have ipam-extensions deploy its dependencies.

Reply:
I think it's more complicated than that. cert-manager by design is not always used. Adding it to the flow may have consequences on other components, so this needs to thought through.

Reply:
While theoretically true, the way the repos work atm (OVN K8s, CNAO, HCO, ipam-ext), on upstream and on OpenShift, should be suitable for deploying cert-manager on ipam-extensions and thats it.
Each component handles it different, and as far as i know it should be fine.

You might want to check deeper, or we can just do a follow-up upon needs.
Note please it might be better to do it on cluster-sync rather than cluster-up, so it will also work for external providers.